user_docs/en/userGuide.md

```diff
@@ -3149,26 +3151,26 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as WordPad, where formatting is important.
 
-##### Spelling error reporting {#ReportSpellingErrors}
+##### Spelling and grammar error reporting {#ReportSpellingErrors}
 
-This option allows you to configure how spelling errors are reported while reading text.
+This option allows you to configure how spelling and grammar errors are reported while reading text.
 This checklist box has three options:
 
-* Speech: NVDA will say "spelling error" when a spelling error is encountered while reading text
-* Sound: NVDA will play a short buzzer sound when a spelling error is encountered while reading text
-When navigating word by word or character by character, "out of spelling error" is also reported if the "Speech" or "Sound" option is selected.
-* Braille: NVDA will indicate spelling errors in braille.
-To have spelling errors reported in braille, [formatting display with tags](#BrailleFormattingDisplayTags) must be enabled.
+* Speech: NVDA will say "spelling error" or "grammar error" when an error is encountered while reading text
+* Sound: NVDA will play a short buzzer or glitch sound when a spelling or grammar error is encountered while reading text
```

Comment:
I think describing the sound in 2 different ways might be confusing - thoughts @Qchristensen ?

```suggestion
* Sound: NVDA will play a short buzzer sound when a spelling or grammar error is encountered while reading text
```

Reply:
I agree, and most users can probably be expected to know what a buzzer sound is, but may also be confused by what a glitch sound is (and whether this describes two distinct sounds).  I agree with your change (to remove "or glitch").